### DISCUSSION TITLE
Global shortcuts for wayland

### DISCUSSION THREAD

**Discussion by alienator** [strawberrymusicplayer/strawberry#1382] [Q&A]
Hi, I am using Gnome with Wayland and I don´t know how to add a shortcut to give love to a song using last.fm, in X11 I can enable the option to use X11 shortcuts and add there the shortcut, that is working fine. There is something similar for wayland?

**Comment by jonaski** [comment]
I think you should be able to configure it in Gnome Settings, provided that it's enabled in strawberry.

**Reply by alienator** [reply]
so, what should be the command, I can see only commands for common tasks like play, stop, etc. But anything for "give love to a song" using last.fm
